### PR TITLE
Adding SLI panel, fixing datasource on version panels

### PR DESCRIPTION
**Pull request by maknop**
- Fixing datasource for versioning with two panels
- Adding SLI panel:
image

### CODE REVIEW

File: dashboards/ephemeral-ns-operator-dashboard.yaml [hunk 1]
```diff
@@ -647,7 +646,7 @@ data:
             {
               "datasource": {
                 "type": "prometheus",
-                "uid": "${datasource}"
+                "uid": "PA6124D83F67FD625"
```
Comment (adamrdrew) [inline dashboards/ephemeral-ns-operator-dashboard.yaml:649]: Should these be fixed data source IDs instead of `${datasource}`?

**Review by adamrdrew — CHANGES_REQUESTED** [review]

File: dashboards/ephemeral-ns-operator-dashboard.yaml (hunk 1 above)
Comment (adamrdrew) [reply dashboards/ephemeral-ns-operator-dashboard.yaml:649]: Got an answer from @maknop - this is all good

**Review by adamrdrew — APPROVED** [review]
LGTM

File: dashboards/ephemeral-ns-operator-dashboard.yaml (hunk 1 above)
Comment (maknop) [reply dashboards/ephemeral-ns-operator-dashboard.yaml:649]: They were set to `${datasource}` so I changed them to be set to the specific datasources required for each panel.

**Pull request merged**